module-services/service-fileindexer/StartupIndexer.cpp

```diff
@@ -20,7 +20,8 @@ namespace service::detail
         using namespace std::string_literals;
 
         // List of initial dirs for scan
-        const std::vector<std::string> start_dirs{purefs::dir::getUserDiskPath() / "music"};
+        const std::vector<std::string> start_dirs{purefs::dir::getCurrentOSPath() / "assets/audio/bell/bg_sounds",
```

Comment:
Do we want to inject product-specific dependencies here? Maybe it would be a better idea to inject that list by using the indexer's constructor?

Reply:
Apart from that, does it work on Pure?

Reply:
IMO, this should passed as parameter

Reply:
Fixed